### ISSUE TITLE
Angular Material select doesn't work

### ISSUE THREAD

**Issue by zbkn** [ag-grid/ag-grid#3317]
**I'm submitting a ...**
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of ag-Grid'
[] support request => see 'Requesting Community Support'
```

**Providing a Reproducible Scenario**
I am using angular components with mat-select in the cell editors. Mat-select isn't changes value, but native select is doing it. [This is an example](https://stackblitz.com/edit/angular-56hunw). It happens on version 21.0.0, on 20.0.0 it is okay.

**Current behavior**
mat-select isn't working

**Expected behavior**
mat-select changes value

**Please tell us about your environment:**
Linux, stackblitz

* **ag-Grid version:** 21.0.0

* **Browser:** Firefox, Chrome

* **Language:** TypeScript 3.1

**Comment by yannickvandervleuten** [comment]
@zbkn could you please explain how you fixed it?

**Comment by zbkn** [comment]
@yannickvandervleuten Hello. I haven't fixed it, I just downgraded version to 20.0.0
